apps/frontend/src/modules/auth/slices/actions.ts

```diff
@@ -48,4 +50,42 @@ const getAuthenticatedUser = createAsyncThunk<
 	return await authApi.getAuthenticatedUser();
 });
 
-export { getAuthenticatedUser, signIn, signUp };
+const sendResetPasswordLink = createAsyncThunk<
+	null,
+	EmailDto,
+	AsyncThunkConfig
+>(`${sliceName}/send-reset-password-link`, async (emailPayload, { extra }) => {
+	const { authApi } = extra;
+
+	try {
+		await authApi.sendResetPasswordLink(emailPayload);
+
+		return null;
```

Comment:
Do you really need to return null here? Seems redundant to me

Reply:
If I don't return null i get this
<img width="1440" alt="Screenshot 2024-08-29 at 10 33 17" src="https://github.com/user-attachments/assets/b15a738d-e69e-402a-ba46-dfeb16832f02">